### DISCUSSION TITLE
How to get image previews of different locations from the API?

### DISCUSSION THREAD

**Discussion by Pkoiralap** [vimeo/player.js#1000] [Feature requests]
Hi,

I am building an annotation tool using VimeoJS and am really struggling with showing a preview image at a certain time in the video. I can see this happening from the player itself using a big image and CSS, but I can't access contents from the iframe and am stuck on this.

Will there be such an API in the future (`getPreviewImage(timestamp)`? For the time being, is there a workaround for this?

Thanks

**Comment by bdougherty** [comment]
I can't say for sure, but I don't think it's likely that those would be made available through the API.
